src/test/java/com/jos/dem/vetlog/service/PetServiceTest.java

```diff
@@ -13,10 +13,7 @@
 import com.jos.dem.vetlog.repository.UserRepository;
 import com.jos.dem.vetlog.service.impl.PetServiceImpl;
 import lombok.extern.slf4j.Slf4j;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.*;
```

Comment:
We normally avoid wildcard imports, so that we can avoid package name collision: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports